### PR TITLE
Task/error handling

### LINKED ISSUE #493: segfault when calling non-existent filter
144 mpi ranks.

### LINKED ISSUE #525: Complain if we are given a actions file and its not there

### PR DESCRIPTION
**Pull request by mclarsen**
resolves #493
resolves #525

### CODE REVIEW

**Comment by mclarsen** [comment]
Don't let errors be rewired when we depend on that behavior

File: src/ascent/ascent.cpp [hunk 1]
```diff
@@ -180,28 +180,39 @@ Ascent::open(const conduit::Node &options)
...
         }
 
+        Node processed_opts(options);
+
+        int comm_id = -1;
+        if(options.has_path("mpi_comm"))
+        {
+          comm_id = options["mpi_comm"].to_int32();
+        }
+
```
Comment (cyrush) [inline src/ascent/ascent.cpp:197]: thanks for adding this check to see if the com is provided.

File: src/ascent/ascent.cpp [hunk 2]
```diff
@@ -371,6 +383,22 @@ Ascent::execute(const conduit::Node &actions)
...
+                // so we better let them know if we don't find
+                // it
+                if(!conduit::utils::is_file(m_actions_file))
+                {
+                    ASCENT_ERROR("An actions file '"
+                                 <<m_actions_file<<"' was specified "
+                                 " but could not be found. Please "
+                                 "check if the file is in the current "
+                                 "directory or provide an absolute path.")
+                }
```
Comment (cyrush) [inline src/ascent/ascent.cpp:400]: thumbs up

**Review by cyrush — APPROVED** [review]
Looks good!

**Pull request merged**